src/hotspot/share/utilities/vmError.cpp

```diff
@@ -675,6 +675,12 @@ void VMError::report(outputStream* st, bool _verbose) {
   // don't allocate large buffer on stack
   static char buf[O_BUFLEN];
 
+  // keep track of which code has already been printed
+  const int printed_capacity = max_error_log_print_code;
+  address printed[printed_capacity];
```

Comment:
Does this buffer get reused/overwritten by the "printing code blobs" logic?

Reply:
The purpose of the buffer is to keep track of what has already been printed. We don't want to print the same code again if we encounter the address again. This can be used across several error reporting steps.

Reply:
Okay, I just wanted to make sure the different usages don't interfere with each other.